[1605] Naujas Django app ir Modelis

Struchid opened the pull request:
- Pridėta nauja Django aplikacija — `uapi`;
    - Aplikacijoje bus saugoma logika naudojantis UDTS specifikacija.
- Pridėtas naujas modelis — `Agent`.
    - Modelis saugos informaciją apie Agentą.
    - Agento kodinis pavadinimas generuojamas iš lauko `title` (pavadinimas) saugojimo metu.

Struchid committed:
New Django app and introduction of Agents

- New django app - UAPI
- New model/concept - Agent

karina-klinkeviciute commented on vitrina/orgs/models.py:
```diff
@@ -336,3 +338,51 @@ class Meta:
+        verbose_name=_("Paskutinės sinchronizacijos data"),
+        blank=True,
+        null=True,
+        help_text=_("Nurodoma data, kada paskutįkart buvo bandyta vykdyti sinchronizaciją."),
```
*paskutinįkart (arba pilnais žodžiais, paskutinį kartą) - ir vertime irgi reiktų pataisyt.

Struchid replied:
Atlikau siūlomus pakeitimus: [4f561e2](https://github.com/atviriduomenys/katalogas/pull/1606/commits/4f561e263d16c3d53d2c78e10024efe1512e9de7)

karina-klinkeviciute commented on vitrina/locale/en/LC_MESSAGES/django.po:
```diff
@@ -3795,6 +3794,41 @@ msgstr "Dislike"
+msgid ""
+"Nurodoma, ar paskutinė sinchronizacija įvyko sėkmingai t.y. jos metu nekilo "
+"klaidų."
+msgstr "Indicates whether the last synchronization was successful, no errors were raised during it"
```
siūlyčiau: msgstr "Indicates whether the last synchronization was successful, i.e. if no errors were raised during it"

Struchid replied:
Atlikau pakeitimus: [4f561e2](https://github.com/atviriduomenys/katalogas/pull/1606/commits/4f561e263d16c3d53d2c78e10024efe1512e9de7)

Struchid committed:
Added new fields for Agent

Struchid commented:
@karina-klinkeviciute @TadasVasiliauskasCCT paprašysiu peržiūros iš naujo, pridėjau papildomų laukų, galite žiūrėti būtent pakeitimų commit'ą: [d1742a5](https://github.com/atviriduomenys/katalogas/pull/1606/commits/d1742a5ce063f7a49a5632159eafc7339ad92ced)

Pagal Manto komentarą:
- https://github.com/atviriduomenys/katalogas/issues/1570#issuecomment-2933592596

martynasracct reviewed (approved):
LGTM

karina-klinkeviciute commented:
atsiprašau, kiek užsiėmus buvau, rytoj peržiūrėsiu

Struchid committed:
[1605] Fixes after Sprint Review

- Changed classification -> object_type
- String representation dunder method added for Agent
- Django admin adjustments for better UX
